src/transformers/models/hubert/configuration_hubert.py

```diff
@@ -115,6 +115,11 @@ class HubertConfig(PretrainedConfig):
             Whether to zero infinite losses and the associated gradients of ``torch.nn.CTCLoss``. Infinite losses
             mainly occur when the inputs are too short to be aligned to the targets. Only relevant when training an
             instance of :class:`~transformers.HubertForCTC`.
+        use_weighted_layer_sum (:obj:`bool`, `optional`, defaults to :obj:`False`):
+            Whether to use a weighted average of layer outputs with learned weights. Only relevant when training an
```

Comment:
```suggestion
            Whether to use a weighted average of layer outputs with learned weights. Only relevant when using an
```
-> do the fine-tuned weights of `s3prl` have this set to ``True``?

Reply:
Yes, true for every task